microservices/product-search-export/.travis.yml

```diff
@@ -3,6 +3,7 @@ language: php
 php:
     - 7.1
     - 7.2
+    - 7.3
```

Comment:
seems like in all travis scripts there is need to put composer install parameter `--ignore-platform-reqs`
see the splitted versions results from travis https://travis-ci.org/shopsys/microservice-product-search-export
i am also no fan of ignoring platform requirements, if the problem is with only one package it would be best to solve problem with that `friendsofphp/php-cs-fixer` rather than to use this kind of param.

Reply:
but is see in their requirements this bug
https://github.com/boris-brtan/PHP-CS-Fixer/blob/8e146bcca71112075ea9bc96791bf975b4873373/composer.json#L17

Reply:
Library updated to a version that has support for 7.3.